.travis.yml

```diff
@@ -107,6 +107,17 @@ jobs:
         - oc login -u developer
         - make test-link-e2e
 
+    # Run json e2e tests
+    - <<: *base-test
+      stage: test
+      name: "json output tests"
+      script:
+        - ./scripts/oc-cluster.sh service-catalog
```

Comment:
is service catalog needed for this tests? If we remove it, it might speed this test up.

Reply:
I noticed it after I pushed, removed it in next commit